code/modules/clothing/under/miscellaneous.dm

```diff
@@ -123,11 +123,11 @@
 
 /obj/item/clothing/under/rank/centcom/blueshield/New()
 	..()
-	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant\" and bears [station_name()] on the left shoulder."
+	desc = "Gold trim on space-black cloth, this uniform bears \"Close Protection\" on the left shoulder."

```

Comment:
This `New()` proc isn't required, now you aren't using a variable in the item's desc